portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -14,19 +14,32 @@
 
 package com.google.sps.servlets;
 
+
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
-
+@WebServlet("/comments")
+public class CommentsServlet extends HttpServlet {
+  /**
+   * Returns the list of all comments
+   */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    CommentsHandler commentsHandler = new CommentsHandler();
+    String comments = commentsHandler.getComments();
+    response.setContentType("application/json");
+    response.getWriter().println(comments);
```

Comment:
I am not exactly familiar with this syntax. Can you please explain here (in a comment) about what is happening?

Reply:
Firstly I get the json array of comments, it is saved to `String comments`, next I set response's content type to json and write comments to the response.

Reply:
Can you add a comment describing this in the code?

Reply:
Sure, I will add more comments